content/en/standard-attributes/_index.md

```diff
@@ -0,0 +1,1807 @@
+    product_source:
+      - icon-apm
+    type: string
+    domain: Network communications
+  - name: network.client.transport
+    description: The transport protocol used to make the inbound connection.
+    product_source:
+      - icon-apm
+    type: string
+    domain: Network communications
+  - name: network.destination.transport
+    description: The transport protocol used to make the outbound connection.
+    product_source:
+      - icon-apm
+    type: string
+    domain: Network communications
+  - name: http.status_code
+    description: The HTTP response status code.
+    product_source:
+      - icon-apm
+    type: string
+    domain: HTTP requests
+  - name: http.url
+    description: The URL of the HTTP request, including the obfuscated query string. For more information on obfuscation, see Configure Data Security.
+    product_source:
+      - icon-apm
+    type: string
+    domain: HTTP requests
+  - name: http.version
+    description: The version of HTTP used for the request.
+    product_source:
+      - icon-apm
+    type: string
+    domain: HTTP requests
+  - name: http.method
+    description: The port of the client that initiated the connection.
+    product_source:
+      - icon-apm
+    type: string
+    domain: HTTP requests
+  - name: http.route
+    description: The matched route (path template). For example, `/users/:userID`
+    product_source:
+      - icon-apm
+    type: string
+    domain: HTTP requests
+  - name: http.client_ip
+    description: The IP address of the original client behind all proxies, if   known. Discovered from headers such as X-Forwarded-For.
+    product_source:
+      - icon-apm
+    type: string
+    domain: HTTP requests
+  - name: http.useragent
+    description: The user agent header received with the request.
+    product_source:
+      - icon-apm
+    type: string
+    domain: HTTP requests
+  - name: http.request.content_length
+    description: The size of the request payload body in bytes.
+    product_source:
+      - icon-apm
+    type: number
+    domain: HTTP requests
+  - name: http.response.content_length
+    description: The size of the response payload body in bytes.
+    product_source:
+      - icon-apm
+    type: number
+    domain: HTTP requests
+  - name: http.request.content_length_uncompressed
+    description: The size of the uncompressed request payload body after   transport decoding.
+    product_source:
+      - icon-apm
+    type: number
+    domain: HTTP requests
+  - name: http.response.content_length_uncompressed
+    description: The size of the uncompressed response payload body after   transport decoding.
+    product_source:
+      - icon-apm
+    type: number
+    domain: HTTP requests
+  - name: http.request.headers.*
+    description: The request HTTP headers. None are collected by default, but can   be optionally configured with DD_TRACE_HEADER_TAGS.To learn more about how to collect   headers, see the corresponding Library configuration.
```

Comment:
I am assuming we can drop the "see the corresponding Library configuration" doc link, feel free to disregard.

```suggestion
    description: The request HTTP headers. None are collected by default, but can be optionally configured with `DD_TRACE_HEADER_TAGS`. 
```